test/unified/runner.js

```diff
@@ -0,0 +1,366 @@
+'use strict';
+const expect = require('chai').expect;
+const path = require('path');
+const semver = require('semver');
+const MongoClient = require('../../src/mongo_client').MongoClient;
+
+class Version {
+  static clean(version) {
+    return semver.coerce(version).version;
+  }
+  static check(supportedSchemaVersions, schemaVersion) {
+    let satisfies = false;
+    supportedSchemaVersions.map(Version.clean).forEach(v => {
+      if (satisfies) return;
+      const result = semver.satisfies(v, `^${Version.clean(schemaVersion)}`);
+      if (result) satisfies = true;
+    });
+    return satisfies;
+  }
```

Comment:
It seems to me that the `Version` class really just wraps a call to `semver.satisfies` with optional support for an array of supported schema versions, is that true? I'm wondering if we need to add this complexity

Reply:
Yep seems true. I don't think it's that complex.